### ISSUE TITLE
Incorrect Application of Alternative Image Text 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#741]
In a number of cases, both alt and aria-label attributes have been applied to <img> elements.  This is unnecessary and causes problems such as: 
 
• Text not being localized.  For example, an “ideas that work” logo image is included in the footer; its alt text is retrieved from a set of localized strings while its aria-label text is hard-coded in English. • Potential for text to become outdated.  If somebody is working on updating alt text for a set of images, they may not realize that the aria-label attribute must also be amended.  As the aria-label value will override any text in the alt attribute, screen reader users will be completely unable to read the new text. 
 
Suggested Solution 
 
Use the alt attribute in all instances.  Remove the aria-label attribute from every <img> in the codebase.

**Comment by benetechMatt** [comment]
on homepage, the mathshare screenshot at the top uses an old alt text and has an aria label
src="images/home-photo.png"

Google classroom and microsoft teams logos have beginning of an alt tag but no value

**Comment by johnhbenetech** [comment]
@rupeshparab My understanding from the recommendation was that NO images should have the aria-label attribute. 

Instead ALL images should use the alt text instead. 

I believe what you've done thusfar is moving the text to the localized strings. But now we need to switch all the aria-labels to alts

**Comment by benetechMatt** [comment]
@johnhbenetech all but that one image use alt now. That's the only image that has an aria label still

**Comment by johnhbenetech** [comment]
@benetechMatt Good call! Just one image then.

I saw some in the social footer icons, but technically those are links not img, and i believe Sina addressed them elsewhere.

**Comment by johnhbenetech** [comment]
@benetechMatt Good call, thanks! Just the one image then.

What I was seeing was the social icons in the footer. Looking closer those are links not img - I think Sina addresses them in a different task.

**Comment by rupeshparab** [comment]
Thanks for pointing it out, I have fixed it

**Comment by jscholes** [comment]
Fiex in https://github.com/benetech/MathShare/commit/4e1a77003db3124b66f3f9556f912ec004bd032f.
